include/ua_client.h

```diff
@@ -65,12 +65,18 @@ typedef void (*UA_ClientStateCallback)(UA_Client *client, UA_ClientState clientS
 
 /**
  * Subscription Inactivity callback
- * ------------------------- */
+ * -------------------------------- */
 
 #ifdef UA_ENABLE_SUBSCRIPTIONS
 typedef void (*UA_SubscriptionInactivityCallback)(UA_Client *client, UA_UInt32 subscriptionId, void *subContext);
 #endif
 
+/**
+ * Inactivity callback
```

Comment:
@StalderT can you add a few sentences to explain the two inactivity callback. I.e. when are the callbacks called, for what can they be used

Reply:
@Pro done in the definition of UA_ClientConfig